invite/src/invite_bot.ts

```diff
@@ -78,5 +172,20 @@ export class InviteBot {
    * Should be called in response to accepted invitations in order to update the
    * thread(s) from which the user was invited.
    */
-  async tryAssign(invite: Invite, accepted: boolean): Promise<void> {}
+  async tryAssign(invite: Invite, accepted: boolean): Promise<void> {
+    const commentStart = accepted ?
+      `The invitation to \`@${invite.username}\` was accepted!` :
+      `It looks like \`@${invite.username}\` is already a member of ` +
```

Comment:
If the invitee is already a member skip this prefix and have the whole comment just say "I've assigned this issue to @username", no need to say that they're already a member in that case

Reply:
Done.